src/peakrdl_sv/reg_top.sv.tpl

```diff
@@ -49,56 +49,28 @@
 
 module ${lblock}_reg_top
   import rdl_subreg_pkg::*;
+  import ${lblock}_reg_pkg::*;
 #(
-  parameter reset_type_e ResetType = ActiveHighSync
+  parameter reset_type_e ResetType = ActiveHighSync,
+  localparam int AW                = ${addr_width},
+  localparam int DW                = ${data_width}
 ) (
   input logic clk,
   input logic rst,
 
-  // Bus I/F
-  // REVISIT: hacked to the Migen CSR bus for now.
-  input logic csr_we,
-  input logic csr_re,
-  input logic [31:0] csr_addr,
-  input logic [7:0] csr_wdata,
-  output logic [7:0] csr_rdata,
+  // CPU I/F
+  input logic          reg_we,
+  input logic          reg_re,
+  input logic [AW-1:0] reg_addr,
+  input logic [DW-1:0] reg_wdata,
+  input logic [DW-1:0] reg_rdata,
```

Comment:
Should be `output logic`

Reply:
I'll take a look at the CI flow now too.